Jamfile

```diff
@@ -5,19 +5,12 @@
 #  Use, modification, and distribution are subject to the
 #  Boost Software License, Version 1.0. (See accompanying file
 #  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+#
 
 project
     : requirements
-
-      <warnings>all
-
-      <toolset>clang:<cxxflags>-Wextra
-      <toolset>clang:<cxxflags>-ansi
-      <toolset>clang:<cxxflags>-pedantic
-
-      <toolset>gcc:<cxxflags>-Wextra
-      <toolset>gcc:<cxxflags>-ansi
-      <toolset>gcc:<cxxflags>-pedantic
+        <warnings>on
```

Comment:
The previous warning level was essentially `<warnings>pedantic`; `<warnings>on` is much less strict.

Reply:
Thanks, I put pedantic back, and we'll see what complains in CI.